pkg/plugin/admissioncontrollers/webhook/admission_controller.go

```diff
@@ -81,9 +96,7 @@ func (c *admissionController) mutate(ar *v1beta1.AdmissionReview) *v1beta1.Admis
 		log.Logger().Warn("request kind is not pod", zap.String("uid", uid),
 			zap.String("requestKind", requestKind))
 
-		return &v1beta1.AdmissionResponse{
-			Allowed: true,
-		}
+		return admissionResponseBuilder(uid, true, "", []byte(""))
```

Comment:
The `[]byte("")` should be replaced with a nil. That lines up with the comment earlier about not adding the patch details if not set.
This should be applied to all calls that pass in the empty array.